src/PrestaShopBundle/Resources/views/Admin/Exception/error.html.twig

```diff
@@ -0,0 +1,76 @@
+{#**
+ * 2007-2019 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% extends '::base.html.twig' %}
+
+{% block stylesheets %}
+  <link rel="stylesheet" href="{{ asset('themes/new-theme/public/theme.css') }}" />
+{% endblock %}
+
+{% block title %}
+  {{ 'Oops... looks like an unexpected error occurred'|trans({},'Admin.Notifications.Error' ) }}
+{% endblock %}
+
+{% block body %}
+  <div class="container">
+    <div class="row mt-5">
+      <div class="col">
+        <div class="card">
+          <div class="card-body text-center">
+            <img class="img-responsive" src="{{ asset('themes/new-theme/img/error/500.svg') }}">
+
+            <div class="mt-3">
+              <p class="error-header">{{ 'Oops... looks like an unexpected error occurred'|trans({}, 'Admin.Notifications.Error') }}</p>
+
+              {% if exception %}
+                <p class="mb-0">{{ 'Error type:' }} {{ exception.class }}</p>
+                <p class="mb-0">{{ 'Error code:' }} {{ exception.code }}</p>
+                <p class="mb-0">{{ 'Error message:' }} {{ exception.message }}</p>
```

Comment:
... and here also! :-D

Reply:
Is this message necessary? I mean, wouldn't it be possible to only write "Enable debug mode and try again to track the issue." without adding "Error message:" before?

Reply:
And I agree with @sarjon, the fact it is centered this way is kind of upseting me... is it a design bias?